hw/ip/flash_ctrl/doc/_index.md

```diff
@@ -405,6 +405,19 @@ The following diagram shows how the flash read pipeline timing works.
 In this example, the first two host requests trigger a full sequence.
 The third host requests immediately hits in the local cache and responds in order after the first two.
 
+### Flash Buffer
+
+The flash buffer is a small read-only memory that holds recently read flash words.
+This is needed when the flash word is wider than a bus word.
+The flash access time is amortized across the the entire flash word if software accesses in a mostly
+linear sequence.
+
+The flash buffer has a round robin replacement policy when more flash words are read.
```

Comment:
So, the flash buffer size exceeds the flash word size?

Reply:
o no i should clarify this.
It's more flash word size > bus word size.
And there are multiple flash words that make up the flash buffer.

Reply:
can you see if the latest explains better @eunchan ?